src/include/distributed/version_compat.h

```diff
@@ -51,8 +49,7 @@
 #define pg_plan_query_compat(p, q, c, b) pg_plan_query(p, c, b)
 #define planner_compat(p, c, b) planner(p, c, b)
 #define standard_planner_compat(a, c, d) standard_planner(a, c, d)
-#define PortalDefineQuerySelectCompat(a, b, c, d, e) PortalDefineQuery(a, b, c, "SELECT", \
-																	   d, e)
+#define CMDTAG_SELECT_COMPAT "SELECT"
```

Comment:
Nice! Like this approach. The name makes it very clear we are going through a compatibility layer.